Sources/NIOFileSystem/FileChunks.swift

```diff
@@ -22,8 +22,8 @@ import NIOPosix
 @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
 public struct FileChunks: AsyncSequence {
     enum ChunkRange {
-        case entireFile
-        case partial(Range<Int64>)
+        case filePointerToEnd
+        case range(Range<Int64>)
```

Comment:
I don't think there's any reason to change these names

Reply:
Isn't `entireFile` misleading because it read from the current file pointer, not from the beginning of the file?